server/service/global_policies.go

```diff
@@ -87,27 +87,32 @@ func (svc Service) NewGlobalPolicy(ctx context.Context, p fleet.PolicyPayload) (
 // List
 /////////////////////////////////////////////////////////////////////////////////
 
+type ListGlobalPoliciesRequest struct {
```

Comment:
Nit: Same as the other request types doesn't need to be exported (should be `listGlobalPoliciesRequest `)